<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_Window</class>
 <widget class="QDialog" name="Room_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Room</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 255, 0);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 0);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 61, 64);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="type">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_room">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>521</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Floor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="floor_room">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1st Floor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2nd Floor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3rd Floor</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="floor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_room">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Single</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Double</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="status_room">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>OK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fixing</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="name_room">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
